packages/components/src/internal/components/entities/EntityMoveConfirmationModal.tsx

```diff
@@ -53,7 +53,7 @@ export const EntityMoveConfirmationModal: FC<EntityMoveConfirmationModalProps> =
                         depth: 1,
                     });
 
-                    const excludedFolders = await api.folder.getDataTypeExcludedProjects(dataType, dataTypeRowId);
+                    const excludedFolders = (!!dataType && !!dataTypeRowId) ? await api.folder.getDataTypeExcludedProjects(dataType, dataTypeRowId) : undefined;
```

Comment:
nit: Feels like logic that should be inside the wrapper if we're going to optimize on the client like this.